packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -3,6 +3,20 @@ title: Okta API Products Release Notes 2021
 ---
 ## June
 
+### Weekly Release 2021.06.1
+
+| Change                                                                         | Expected in Preview Orgs |
+|--------------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2021.06.1](#bugs-fixed-in-2021-06-1)                            | June 9, 2021             |
+
+#### Bugs fixed in 2021.06.1
+
+* User credential updates failed and returned an error message if the [User Type](/docs/reference/api/user-types/) was set to a non-email username. (OKTA-334754)
+
+* When an [authorize](/docs/reference/api/authn/) request with an [IdP param](/docs/reference/api/idps/#add-identity-provider) that referenced a [SAML IdP](/docs/reference/api/idps/#add-saml-2-0-identity-provider) was sent, an error message was returned because the OIDCIdpAuthRequestHelperImpl request assumed that all IdPs are Social IdPs. (OKTA-385800)
```

Comment:
I went back to Miao and Graham with this one - because I don't feel that including the method name helps with clarity here. Also, too many links included. I would suggest just a link to the actual IdP parameter page, since that was part of the issue. This blurb has been approved by Miao and Graham and updated in the edit spreadsheet
```suggestion
* When an `/authorize` request with an [IdP parameter](/docs/reference/api/idps/#add-identity-provider) that referenced a SAML Identity Provider (IdP) was sent, an internal server error was returned because it was assumed that all IdPs are social IdPs. (OKTA-385800)
```